central/imageintegration/datastore/datastore_impl_test.go

```diff
@@ -226,6 +236,10 @@ func (suite *ImageIntegrationDataStoreTestSuite) TestAllowsGetBatch() {
 	gotImages, err = suite.datastore.GetImageIntegrations(suite.hasWriteCtx, getRequest)
 	suite.NoError(err, "expected no error trying to read with permissions")
 	suite.ElementsMatch(integrationList, gotImages)
+
+	gotImages, err = suite.datastore.GetImageIntegrations(suite.hasWriteIntegrationsCtx, getRequest)
+	suite.NoError(err, "expected no error trying to read with Integrations permissions")
+	suite.ElementsMatch(integrationList, gotImages)
 }
 
 func (suite *ImageIntegrationDataStoreTestSuite) TestEnforcesAdd() {
```

Comment:
Should there be tests with suite.hasWriteIntegrationsCtx for Add, Update and Remove ?